components/molecules/PullRequestOverview/pull-request-overview.tsx

```diff
@@ -12,58 +11,33 @@ interface PullRequestOverviewProps {
   churnDirection?: string;
   prActiveCount?: number;
 }
+interface OverviewDetails {
+  type: string;
+  percent: number | undefined;
+}
 
-const PullRequestOverview: React.FC<PullRequestOverviewProps> = ({
-  className,
-  open,
-  merged,
-  closed,
-  draft,
-  churn,
-  churnDirection = "down",
-  prActiveCount,
-}) => {
+const PullRequestOverview: React.FC<PullRequestOverviewProps> = ({ open, merged, closed, draft, prActiveCount }) => {
   const totalPullRequests = Math.round(
     (!!open ? open : 0) + (!!merged ? merged : 0) + (!!closed ? closed : 0) + (!!draft ? draft : 0)
   );
-  const prCount = prActiveCount || 0;
-  const activePrPercentage = totalPullRequests > 0 ? Math.round((prCount / totalPullRequests) * 100) : 0;
-  // used this state to manage actively hovered pull request overview chart
-  interface OverviewDetails {
-    type: string;
-    percent: number | undefined;
-  }
 
-  const [prOverviewDetails, setPrOverviewDetails] = useState<OverviewDetails>({ type: "closed", percent: closed });
+  // used this state to manage actively hovered pull request overview chart
+  const [prOverviewDetails, setPrOverviewDetails] = useState<OverviewDetails>({
+    type: "",
```

Comment:
should the type be an empty string by default? The types appear to be "open" "closed" and "merged" only.

Reply:
The default state is the total prs... `merged` + `open` + `closed` + `draft`. And that doesn't require a type prefix. I'll adjust the type to be explicitly defined as suggested with an empty string to it.